src/rastervision/protos/train.proto

```diff
@@ -17,6 +17,25 @@ message TrainConfig {
         optional string export_py = 3 [default="/opt/tf-models/object_detection/export_inference_graph.py"];
     }
 
+    message SegmentationOptions {
```

Comment:
It seems like these parameters should be in a backend config file which is referenced from the backend_config_uri field to be consistent with the rest of RV. This would involve moving the bulk of the fields in SegmentationOptions into a DeepLabBackendConfig proto or similar.